xds/src/test/java/io/grpc/xds/orca/OrcaServiceImplTest.java

```diff
@@ -125,7 +123,8 @@ public void testReportingLifeCycle_serverShutdown() {
     OrcaLoadReport expect = OrcaLoadReport.newBuilder().putUtilization("buffer", 0.2).build();
     assertThat(defaultTestService.getClientsCount()).isEqualTo(1);
     verify(listener).onMessage(eq(expect));
-    reset(listener);
+    verify(listener).onHeaders(any(Metadata.class));
```

Comment:
Why add these two lines? It seems they aren't necessary. I can understand `onHeaders()` to make it similar to other tests, but I don't think we should be using verifyNoMoreInteractions() willy-nilly.